src/sdk/namespace_fs.js

```diff
@@ -333,7 +333,14 @@ class NamespaceFS {
             const file_path = this._get_file_path(params);
             const stat = await fs.promises.stat(file_path);
             console.log(file_path, stat);
-            return this._get_object_info(params.bucket, params.key, stat);
+            if (stat.isDirectory()) {
+                const err = new Error('NoSuchKey');
+                err.code = 'ENOENT';
+                throw this._translate_object_error_codes(err);
+            }
+	    else {
+	      return this._get_object_info(params.bucket, params.key, stat);
+            }
```

Comment:
something is off with indent - make sure your replace tabs with 4 spaces.
Also since we throw just before and now have made sure the rest of the flow is only for files, we can ditch the else clause:
```suggestion
            return this._get_object_info(params.bucket, params.key, stat);
```

Reply:
I thought I pushed the changes you recommended but I don't see them here, I must have made a mistake

Reply:
still failing some test

Reply:
the "Unit Tests with Postgres" failures should be ignore for now.
right @liranmauda @nimrod-becker ?

Reply:
@guymguym correct
@nimrod-becker @dannyzaken we might want to disable the tests that we know are failing, so we will not ignore the whole suit, and we will actually know if something was broken.

Reply:
Resolved in #6418 so this failure will be gone once you rebase from master.